### PR TITLE
Taking a stab at a blood mite update

### PR DESCRIPTION
**Pull request by izawwlgood**

### CODE REVIEW

**Comment by izawwlgood** [comment]
This is what they look like when on you, hopefully the syntax is shared with leeches - 

> You have a small red blood mite on your right leg, a small red blood mite on your right arm.

**Comment by jandersson** [comment]
Theres a helper function called `check_health` that you can use to find the leeches and their locations.
`DRCH.check_health['parasites']`

**Comment by izawwlgood** [comment]
Tested this out, works acceptable but has some unintended behavior - 

> >;tend
> [tendme]>heal
> Your body feels slightly battered.
> Your spirit feels full of life.
> You have some minor abrasions to the right arm, some minor abrasions to the left arm, some minor abrasions to the right leg.
> 
> You have a small red blood mite on your right arm, a small red blood mite on your left arm.
> >
> [tendme]>tend my right arm
> You deftly remove the blood mite from your right arm leaving the wound no worse than it was before.
> 
> The blood mite slips free and quickly slithers away, vanishing from sight within moments.
> Roundtime: 10 seconds.
> >
> [tendme]>tend my right arm
> That area is not bleeding.
> >
> [tendme]>tend my left arm
> You deftly remove the blood mite from your left arm leaving the wound no worse than it was before.
> 
> The blood mite slips free and quickly slithers away, vanishing from sight within moments.
> Roundtime: 10 seconds.
> >
> [tendme]>tend my left arm
> That area is not bleeding.
> >
> [tendme]>tend my right arm
> That area is not bleeding.
> >
> [tendme]>tend my left arm
> That area is not bleeding.

**Comment by jandersson** [comment]
Can you post more of the log?

**Comment by jandersson** [comment]
Its because your do a loop on 'message' for mites and leeches

**Comment by jandersson** [comment]
The lines for `mites.each` and `leeches.each` are matching on the same thing inside the loop, so any match for 'leeches' also happens in the 'mites' loop and viceversa

File: tendme.lic [hunk 1]
```diff
@@ -38,6 +43,11 @@ def bind_open_wounds?
     DRCH.bind_wound(Regexp.last_match(1))
   end
 
+  mites.each do |message|
+    message =~ /on your ([^\,\.]+)|(\,|\.)/
```
Comment (jandersson) [inline tendme.lic]: This matches line 47

File: tendme.lic [hunk 2]
```diff
@@ -38,6 +43,11 @@ def bind_open_wounds?
...
+    DRCH.bind_wound(Regexp.last_match(1))
+  end
+
   leeches.each do |message|
     message =~ /on your ([^\,\.]+)|(\,|\.)/
```
Comment (jandersson) [inline tendme.lic]: This matches line 47

Comment (izawwlgood) [reply tendme.lic]: Should I change to - 

>   mites.each do |message|
>     message =~ /mite on your ([^\,\.]+)|(\,|\.)/
>     DRCH.bind_wound(Regexp.last_match(1))
>   end
> 
>   leeches.each do |message|
>     message =~ /leech on your ([^\,\.]+)|(\,|\.)/
>     DRCH.bind_wound(Regexp.last_match(1))
> end

?

Comment (jayRyan24) [reply tendme.lic]: @jandersson

**Pull request closed without merging**